.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 static/
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+*.rdb
```

Comment:
Git kept trying to commit a dump.rdb file, so I added this to .gitignore.  I can remove it if it is a problem.

Reply:
no problems!